tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+set(STL_SOURCE_DIR "${PROJECT_SOURCE_DIR}")
+set(STL_BUILD_ROOT "${PROJECT_BINARY_DIR}/out")
+set(STL_TESTED_HEADERS_DIR "${STL_BUILD_ROOT}/inc")
+set(STL_TEST_UTILS_DIR "${CMAKE_CURRENT_SOURCE_DIR}/utils")
+set(STL_TEST_OUTPUT_DIR "${PROJECT_BINARY_DIR}/test")

```

Comment:
could just use ${CURRENT_BINARY_DIR} instead.

Reply:
I actually don't use that variable anywhere anymore so I can drop it.

Reply:
Hrm, on second thought, I should make everything respect it so I can repoint the root of where the tests are run easily.